### DISCUSSION TITLE
Feature Request(s) 😇

### DISCUSSION THREAD

**Discussion by nsk88** [fastaddons/GroupSpeedDial#361] [Ideas]
[2024-11-28] 

For your consideration:

(1) A "Search Settings" feature to exist below the "Search" function in the options panel which would allow a person to quickly find a setting based on a search string. E.g. Firefox or Chrome allow you to search through the settings with a search term. GroupSpeedDIal's "Search" function does a great job searching through groups and dials. Just wanting the same functionality for settings. :)

(2) An "Enable Column Names" and "Enable Row Names" setting to allow the labeling of columns and rows within a box. Ability to place these boxes at top or bottom for columns, and left or right for rows. Also, a setting to change height (column) or width (row) of those boxes. You could then also allow a function for right clicking these column or row boxes to "Open All Dials from this Column in new tabs" and "Open All Dials from this Row in new tabs". This might help greatly for fine-tuning groups by categories, and loading by category.

Thank you for your continued support of this great addon.
Cheers!

**Comment by Juraj-Masiar** [comment]
Hello,
Great ideas!
The search settings is a bit tricky. I was thinking about that few times, but I couldn't think of a good solution.
I'll put it on my todo list and look into that when I migrate to Vue 3.
Maybe if rendered all Options tabs below each other, I could perform the search, or even let user use the build-in browser search (but that would be intuitive enough). I'll think about it!

The column/row names was already on my todo list, but the new idea is actually "named groups", so that you wouldn't be limited by rows/column, but you could select even for example "4 dials in the corner". This would still allow you to create rows/columns though.

I'll see what I can do. These and many other good ideas are on my long todo list. I'll do my best to implement it in the near future :).

**Reply by nsk88** [reply]
Thank you so much! Very impressed by your work! =)

**Comment by BrettieG** [comment]
Great up, used to use another for many years but your is stable, better and just fantastic. Thank you.
What would be great feature don't know if Pro has it, it is not listed in the specs from what I briefly saw, is when one exports the dials to be able to select which Tabs one would like exported or backed up. Default One can then do full back, but for sharing of dials across a team environment, you don't have to import unnecessary tab's. For privacy it would be more beneficial to do this on the export than on the import of selective tabs. Hope this make sense.

**Reply by BrettieG** [reply]
When I say Tab's , you refer to as Groups.

**Reply by Juraj-Masiar** [reply]
There is a "Share dials" feature in the main menu.
I'm still missing documentation for that, but I should get to that soon.
In any case, this feature allows you to create a sharable link of dials / groups, and people that opens your shared link will be able to import it, for example:
https://group-speed-dial.fastaddons.com/cloud/?share=2CqEJ2MiQEi_HnLR2tgTag

**Comment by mikwei** [comment]
Search - like the search (never used til today).   But this and several other actions are used often(or not).  But an option icon in menu-bar for commonly used items vs hunting.  So something needing a menu to add items to a quick action list.

Just a thought, but think I will use search now more often.

**Comment by mikwei** [comment]
Would like to be able to show thumbnail from 1 site, but click takes me elsewhere.   Thinking for weather where 1 site(local) gives current info(thumbnail), but clicking sends me to a different site with forecasts...

Custom Thumbnail from URL does not seem to work, am pointing to a local url where link points outwards.
